frontend/src/containers/AddChart.tsx

```diff
@@ -198,8 +198,8 @@ function AddChart() {
                 hint={
                   <span>
                     Must be a CSV file.{" "}
-                    <Link to="/admin/formattingcsv" target="_blank" external>
-                      How do I format my CSV file?
+                    <Link to="/admin/apihelp" target="_blank" external>
+                      How do I add datasets?
```

Comment:
are we removing the link of "How do I format my CSV file?"

Reply:
The wire frame doesn't show it: https://aws-uxdr.invisionapp.com/share/QXVZZ6VE6F7#/screens. But this whole section will need to redone as part of the addition of static and dynamic datasets. So adding the link seemed like it might save someone a step later 🤷‍♀️

Reply:
You can remove the new link and leave it as it is now, I'll add it as part of the datasets story

Reply:
Sounds good.